### ISSUE TITLE
ML-Tutorial-Experiment/Experiments/tf_LeNet5.ipynb

### ISSUE THREAD

**Issue by huangnengCSU** [jiqizhixin/ML-Tutorial-Experiment#5]
pool1 = tf.nn.max_pool(relu1, ksize=[1, 2, 2, 1],strides=[1,1,1,1], padding='SAME')
the strides should be [1,2,2,1] instead of [1,1,1,1]
input:28×28×32
output:14×14×32

**Comment by HoratioJSY** [comment]
thanks, we have fixed~
